Objects/moduleobject.c

```diff
@@ -1144,10 +1145,97 @@ module_get_annotations(PyModuleObject *m, void *Py_UNUSED(ignored))
         Py_DECREF(dict);
         return NULL;
     }
+    return dict;
+}
+
+static PyObject *
+module_get_annotate(PyModuleObject *m, void *Py_UNUSED(ignored))
+{
+    PyObject *dict = module_get_dict(m);
+    if (dict == NULL) {
+        return NULL;
+    }
+
+    PyObject *annotate;
+    if (PyDict_GetItemRef(dict, &_Py_ID(__annotate__), &annotate) == 0) {
+        annotate = Py_None;
+        int result = PyDict_SetItem(dict, &_Py_ID(__annotate__), annotate);
+        if (result == -1) {
```

Comment:
```suggestion
        if (PyDict_SetItem(dict, &_Py_ID(__annotate__), annotate) == -1) {
```

I'm not sure if you want to keep the result separately for clarity (I'd say half of your `PyDict_SetItem` use an intermediate result).

Reply:
You're right, I don't need it here, will change this.